### DISCUSSION TITLE
Change theme

### DISCUSSION THREAD

**Discussion by ochilliam** [nicknisi/dotfiles#217] [General]
Hi @nicknisi 
I've been seeing this theme (not sure a theme or bug) in my vim.
text are not clear to read and it's ugly. could you help?
one more thing, I don't seem to be able to change theme with base_16 like I used to, what's the issue?

<img width="1361" alt="CleanShot 2023-09-22 at 09 50 26@2x" src="https://github.com/nicknisi/dotfiles/assets/65443737/ce4e5e2b-1350-425b-9511-5db4e3257c94">

**Comment by nicknisi** [comment]
Hey @ochilliam! So, I've actually moved away from base16 in favor of [Catppuccin](https://github.com/catppuccin/catppuccin). If using the latest branch, it should automatically install and configure. However, it's likely depending on the underlying terminal emulator you're using to provide the background coloring. This is configured on this line:

https://github.com/nicknisi/dotfiles/blob/6e48bf4205290874480e8901894aeb0e0c99b8b0/config/nvim/lua/plugins/ui.lua#L351

Changing `transparent_background` to `false` should make the neovim plugin provide the right background coloring.

Alternatively, you can use a terminal emulator like [Wezterm](https://wezfurlong.org/wezterm/index.html) or [Kitty](https://sw.kovidgoyal.net/kitty/) with the provided configs to provide the background coloring through the terminal.

Hope this helps!

**Comment by ochilliam** [comment]
@nicknisi I'm using Kitty with your config. my setup is basically your setup
could you please help

**Reply by nicknisi** [reply]
Ah ok. Is it possible that you don't have the right theme installed for Kitty? Try running 

```bash
kitty +kitten themes
```

And select `Catppuccin-Mocha`.

![capture-20230927084411](https://github.com/nicknisi/dotfiles/assets/293805/fcd0f363-f6cc-4bf1-9ed8-93e4e0baca85)

If that doesn't solve it, try running the following commands from the dotfiles repo

```bash
tic -x resources/xterm-256color-italic.terminfo 
tic -x resources/tmux.terminfo 
```

Then, restart Kitty and see if it fixes. Hope this helps!

**Comment by ochilliam** [comment]
@nicknisi thanks, didn't help. tried both of them.

I'll get deeper to see I can solve it. could you send a screenshot of how it should look like?

**Reply by nicknisi** [reply]
It looks like this in Kitty.

![capture-20230927095026](https://github.com/nicknisi/dotfiles/assets/293805/56638d1f-0298-411f-a7ba-c8eee1084d08)

Alternatively, you could try [WezTerm](https://wezfurlong.org/wezterm/index.html) and see if you have better luck with that? This is what it looks like for me, and the configuration is much more contained.

![capture-20230927095145](https://github.com/nicknisi/dotfiles/assets/293805/beacd59f-7d35-4bb4-a86c-c10e99bc42d1)

**Comment by ochilliam** [comment]
Ok fixed. They look gorgeous. 

did a clean `install all`. my system theme was light, I think that was part of the issue.

[WezTerm](https://wezfurlong.org/wezterm/index.html) is awesome but upon fast navigation (e.g vim up and down) it isn't smooth as kitty is.


https://github.com/nicknisi/dotfiles/assets/65443737/1d9771a7-940b-4dd4-99d1-ebc005b5ebee
